syntax = "proto3";
package LN_Chat;

service ChatService {
  rpc PublishRoom(PublishRoomRequest) returns (PublishRoomResponse);
}

message PublishRoomRequest {
    string name = 1;
    string password = 2;
}

message PublishRoomResponse {
    bool success = 1;
    string message = 2;
}

